src/asyncapidiff.ts

```diff
@@ -17,11 +18,13 @@ import toProperFormat from './helpers/output/toProperFormat';
 export default class AsyncAPIDiff {
   private output: JSONOutput;
   private outputType: OutputType;
+  private markdownSubtype: MarkdownSubtype;
 
   constructor(output: string, options: AsyncAPIDiffOptions) {
     // output is a stringified JSON
     this.output = JSON.parse(output);
     this.outputType = options.outputType;
+    this.markdownSubtype = options.markdownSubtype || 'json';
```

Comment:
In our current implementation, we ensure that it's JSON. But then, to be able to remove the `|| 'json'` we'll need to either make `markdownSubtype` a required property in AsyncAPIDIffOptions or allow `this.markdownSubtype` to be undefined, otherwise the types don't match.